src/components/templates/t-discover.vue

```diff
@@ -88,6 +86,15 @@ export default {
     searchedText: String,
     show: Boolean,
   },
+  computed: {
+    place() {
+      const temp = this.searchedText.indexOf("/");
```

Comment:
please add some sample searchText in comments

Reply:
done